Tuist/ProjectDescriptionHelpers/Target+Helpers.swift

```diff
@@ -26,10 +26,10 @@ extension Target {
         }
         return Target(
             name: name,
-            destinations: [.mac],
```

Comment:
Just curious what the failure was here.  the `platform` initializer is going to be removed with Tuist 4.